docs/server/source/simple-network-setup.md

```diff
@@ -0,0 +1,197 @@
+
+```bash
+bigchaindb configure
+```
+
+The first question is ``API Server bind? (default `localhost:9984`)``, to expose the API to the public, bind the API Server to `0.0.0.0:9984`. Unless you have specific needs, you can keep the default value for all other questions.
+
+## Member: Generate the private key and address
+A Node is identified by the triplet `<hostname, address, public_key>`.
+
+As a Member, it's your duty to create and store securely your private key, and share your `hostname`, `address`, and `public_key` with the other members of the network.
+
+To generate all of that, run:
+
+```bash
+tendermint init
+```
+
+The `public_key` and `address` are stored in the file `.tendermint/config/priv_validator.json`, and it should look like:
+
+```json
+{
+  "address": "5943A9EF6285791A504918E1D117BC7F6A615C98",
+  "pub_key": {
+    "type": "AC26791624DE60",
+    "value": "W3tqeMCU3d4SHDKqrwQWTahTW/wpieIAKZQxUxLv6rI="
+  },
+  "last_height": 0,
+  "last_round": 0,
+  "last_step": 0,
+  "priv_key": {
+    "type": "954568A3288910",
+    "value": "3sv9aExgME6MMjx0JoKVy7KtED8PBiPcyAgsYmVizslbe2p4wJTd3hIcMqqvBBZNqFNb/CmJ4gAplDFTEu/qsg=="
+  }
+}
+```
+
+Share with all other Members your `address`, `pub_key.value`, and the host name or IP of your Node.
+
+**Important note on security: each Member should verify the data they receive from the other Members.**
+
+## Coordinator: Initialize the Network
+At this point the Coordinator should has received the data from all the Members, and should combine them in the `.tendermint/config/genesis.json` file:
+
+```json
+{
+  "genesis_time": "0001-01-01T00:00:00Z",
+  "chain_id": "test-chain-la6HSr",
+  "validators": [
+    {
+      "pub_key": {
+        "type": "AC26791624DE60",
+        "value": "<Member 1 public key"
+      },
+      "power": 10,
+      "name": "<Member 1 name>"
+    },
+    {
+     "pub_key": {
+       "type": "AC26791624DE60",
+       "value": "<Member 2 public key>"
+     },
+     "power": 10,
+      "name": "<Member 2 name>"
+    },
+    {
+     "...": { },
+    },
+    {
+     "pub_key": {
+       "type": "AC26791624DE60",
+       "value": "<Member N public key>"
+     },
+     "power": 10,
+      "name": "<Member N name>"
+    }
+  ],
+  "app_hash": ""
+}
+```
+
+The new `genesis.json` file contains the data that describes the Network.
+
+At this point, the Coordinator must share the new `genesis.json` file with all Members.
+
+## Member: Connect to the other Members
+At this point the Member should have received the `genesis.json` file.
+
+**Important note on security: each Member should verify that the `genesis.json` file contains the correct public keys.**
+
+The Member must copy the `genesis.json` file in the local `.tendermint/config` directory.
+
+The Member must edit the `.tendermint/config/config.toml` file and make the following changes:
+
+```
+...
+
+create_empty_blocks = false
+...
+
+persistent_peers = "<Member 1 address>@<Member 1 hostname>:46656,\
```

Comment:
It would be good to mention that the member address or member node id can be generated using `tendermint show_node_id`

Reply:
Thanks, I should change `address` to `node id`.